src/researches/passiveVoice/morphological/determinePassiveSentence.js

```diff
@@ -0,0 +1,49 @@
+const filter = require( "lodash/filter" );
+const getWords = require( "../../../stringProcessing/getWords.js" );
+
+// Verb-form lists per language
+const getPassiveVerbsRussian = require( "../../russian/passiveVoice/participlesShortenedList.js" )().all;
+
+/**
+ * Matches the sentence against passive verbs.
+ *
+ * @param {string} sentence The sentence to match against.
+ * @param {Array} passiveVerbs The array containing passive verb-forms.
+ * @returns {Array} The found passive verbs.
+ */
+let matchPassiveVerbs = function( sentence, passiveVerbs ) {
+	return filter( passiveVerbs, function( word ) {
+		let sentenceWords = getWords( sentence );
+		return sentenceWords.includes( word );
```

Comment:
To me it makes more sense to call `sentenceWords` `words`, and `word` `verb`. What do you think?

Reply:
Agreed, but following the suggestion of Anton on matching the other way around it now makes more sense to keep `word` as `word` (a word of a sentence). And I got rid of the sentenceWords array altogether.